packages/core/src/collection.ts

```diff
@@ -194,10 +194,28 @@ export function collection<T = any>(name: string, { idField = 'id' }: Collection
     return changeFunctions
   }
 
+  /**
+   * Takes an updated version of the given doc (can be partial); returns a function merging the
+   * updates with the existing doc, mapped to the collection key corresponding to the id field in
+   * item.
+   * @param item An object containing the id of the item to modify, as well as the values to be
+   * modified. This can be either a modified copy of the whole document, or just the fields to be
+   * changed.
+   */
   const update = (item: A.Doc<any>) => ({
     [idToKey(item[idField])]: (s: any) => Object.assign(s, item),
   })
 
+  /**
+   * Takes a change function and an id, and returns the function mapped to the collection key.
+   */
+  const change = (id: string, fn: ChangeFn<T>) => ({
+    [idToKey(id)]: fn,
+  })
```

Comment:
In which cases should this be used instead of `update`?

Reply:
In situations where you can't provide the replacement object, just a change function. This test for example: https://github.com/DevResults/cevitxe/blob/fix-todo/packages/core/src/StoreManager.test.ts#L146-L166

I'm really dissatisfied with this whole API, as per #40, and the whole collection reducer/selector thing is a big part of what feels janky to me. So I wouldn't fixate on this too much because it all needs to change anyway.